src/components/dropdown-filter/dropdown-filter.js

```diff
@@ -341,7 +346,11 @@ class DropdownFilter extends Dropdown {
           className='carbon-dropdown__action'
           data-element='create'
           key='dropdown-action'
+          onBlur={ this.handleCreateBlur }
           onClick={ this.handleCreate }
+          onKeyPress={ this.handleCreate }
+          role='button'
+          tabIndex='0'
```

Comment:
changing this to an actual button would mean you don't need `onKeyPress`, `role`, or `tabIndex`.

Reply:
Changing to a native `<button>` does mean we don't need to implement those additional attributes. However, the default behaviour when focus is switched from the input (using the tab key) is to close the dropdown.